pkg/providers/v1/instances_v2.go

```diff
@@ -74,9 +79,32 @@ func (c *Cloud) getAdditionalLabels(zoneName string) (map[string]string, error)
 
 	additionalLabels[LabelZoneID] = zoneID
 
+	if c.topologySupportedInstance(instanceType) && slices.Contains(c.cfg.Global.TopologySupportedRegions, region) {
+		topologyRequest := &ec2.DescribeInstanceTopologyInput{InstanceIds: []*string{&instanceID}}
+		topology, err := c.ec2.DescribeInstanceTopology(topologyRequest)
+		if err != nil || topology == nil {
+			klog.Infof("topology api not supported for instance type: %s or region: %s", instanceType, region)
+			return additionalLabels, nil
```

Comment:
As we will use the supportedInstanceType instead of instance prefix now, the error throw our there should not because not supported issue. I throw the error directly here in new version